### DISCUSSION TITLE
Can't consume liquids right after the craft?

### DISCUSSION THREAD

**Discussion by Foxtrot-Uniform42** [CleverRaven/Cataclysm-DDA#46704] [Q&A] [answered]
I've noticed that option to immediately apply/consume substances just after the craft is missing (requires a container now). I hadn't found any PR's or issues related to it, is this intended?
![image](https://user-images.githubusercontent.com/46412513/104391594-04c35c00-5562-11eb-8909-0a9f19673a89.png)

**Comment by anothersimulacrum** [comment, marked as answer]
Yes. Because things take time to consume, items must be contained or the game cannot proceed.

This was introduced in #40975
